packages/devextreme/js/ui/card_view.d.ts

```diff
@@ -0,0 +1,1880 @@
+} from '../common/grids';
+import DataSource, { DataSourceLike } from '../data/data_source';
+import Widget, { WidgetOptions } from './widget/ui.widget';
+import { Cancelable, EventInfo, NativeEventInfo } from '../events';
+import { dxToolbarItem, ToolbarItemLocation } from './toolbar';
+import { dxSortableOptions } from './sortable';
+import { dxLoadPanelOptions } from './load_panel';
+import dxScrollable from './scroll_view/ui.scrollable';
+import {
+    Properties as PopupProperties,
+  } from './popup';
+import {
+    Properties as FormProperties, SimpleItem,
+  } from './form';
+import { dxFilterBuilderOptions } from './filter_builder';
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type Paging = {
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    enabled?: boolean;
+    /**
+     * @docid
+     * @default 0
+     * @fires dxCardViewOptions.onOptionChanged
+     * @public
+     */
+    pageIndex?: number;
+    /**
+     * @docid
+     * @default 6
+     * @fires dxCardViewOptions.onOptionChanged
+     * @public
+     */
+    pageSize?: number;
+};
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type RemoteOperations = {
+    /**
+     * @docid
+     * @default false
+     */
+    filtering?: boolean;
+    /**
+     * @docid
+     * @default false
+     */
+    paging?: boolean;
+    /**
+     * @docid
+     * @default false
+     */
+    sorting?: boolean;
+    /**
+     * @docid
+     * @default false
+     */
+    grouping?: boolean;
+};
+
+// #region Toolbar
+
+/** @public */
+export type PredefinedToolbarItem = 'columnChooserButton' | 'searchPanel' | 'addCardButton' | 'selectAllButton' | 'clearSelectionButton';
+
+/**
+ * @docid
+ * @inherits dxToolbarItem
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type ToolbarItem = dxToolbarItem & {
+    /**
+     * @docid
+     * @public
+     */
+    name?: PredefinedToolbarItem | string;
+    /**
+     * @docid
+     * @default 'after'
+     * @public
+     */
+    location?: ToolbarItemLocation;
+};
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type Toolbar = {
+    /**
+     * @docid
+     * @public
+     */
+    items?: Array<PredefinedToolbarItem | ToolbarItem>;
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    visible?: boolean | undefined;
+    /**
+     * @docid
+     * @default false
+     * @public
+     */
+    disabled?: boolean;
+    /**
+     * @docid
+     * @default false
+     * @public
+     */
+    multiline?: boolean;
+};
+
+// #endregion
+
+// #region ColumnsController
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type FieldInfoType = { // TODO: rename to FieldInfo
+    /**
+     * @public
+     * @docid
+     * @type object
+     */
+    value: unknown;
```

Comment:
Not 100% sure about that, but it seems that we shouldn't use `unknown` for values that can be passed to consuming code. It's generallly a good practise, but all the other components currently use `any`.

```suggestion
    value: any;
```

Reply:
Agree with u here
Let's not risk and use ```any``` for now

Btw, @IlyaKhD, what do u think about ```unknown``` in generics?
I guess it should be alright, but at night it started worried me a bit because ```unknown``` !== ```any```

Reply:
Well, I don't have a general idea for all generics, better investigate some samples. We have reasons to avoid giving unknown outside, and can leverage from marking as unknown for data going inside. Is there a gurantee that generics never given outside - hm... not sure about that, it will be hard to lint, I think. Not sure about possible covariance / contravariance issues, 2-way bindings in Angualr, etc